src/json-schema-view.js

```diff
@@ -142,6 +143,10 @@ export default class JSONSchemaView {
             <span class="range maxLength">maxLength:${this.schema.maxLength}</span>
           `}
 
+           ${_if(!this.isCollapsed && this.schema.pattern)`
+            <span class="range pattern">pattern:${this.schema.pattern}</span>
```

Comment:
Do you want to add some css to this?

Reply:
Didn't have any specific CSS requirements in mind - i added the class just in case....
If my UX designer suddenly throws some requirements at me I'll put in another PR if thats OK?

Reply:
Yes, we want tot make sure this is not a value and is just a modifier like range and others.

Reply:
Not sure I follow. Do I need to add some additional checks?

![image](https://user-images.githubusercontent.com/7387884/45843176-c5c8c700-bd16-11e8-82d2-11e9db63cdf0.png)

Thats how it currently looks. Which is good enough for my purposes.